src/Symfony/Bridge/Doctrine/Security/User/UserLoaderInterface.php

```diff
@@ -29,4 +29,8 @@
  */
 interface UserLoaderInterface
 {
+    /**
+     * @deprecated since Symfony 5.3, use loadUserByIdentifier() instead
```

Comment:
The phpdoc documenting the return type is missing

Reply:
Thanks! Fixed.